rj_utils/include/rj_utils/logging.hpp

```diff
@@ -0,0 +1,189 @@
+#pragma once
+
+/**
+ * \file logging.hpp
+ * \brief This file defines macros that wrap the ones in rclcpp/logging.hpp
+ * but with a NOLINT at the end, so that we don't trip up clang_tidy every time
+ * we make a logging call. This also defines EZ prefixed log calls that
+ * automatically use get_logger() to get the current logger, so that logging
+ * calls can be shorter.
+ */
```

Comment:
I would add the link to the log stuff since there's tons of extra functions. Something like...

```suggestion
 * See link for log descriptions  http://docs.ros2.org/foxy/api/rclcpp/logging_8hpp.html
 */
```